receipt/SaleReceipt.tpl

```diff
@@ -845,7 +901,7 @@ table.payments td.label {
 		{% endif %}
 
 		{% if Sale.Customer %}
-			{% if Sale.Customer.company|strlen > 0 %}
+			{% if Sale.Customer.company|strlen > 0 and not options.show_customer_name_only %}
```

Comment:
There seems to be some additional updates beyond what is required for this cycle, is that intended?

Reply:
Will we need these changes to our public files? https://github.com/merchantos/webPOS/commit/a93700b6e037ed7629d7f2e95090159318baf29c

Reply:
Potentially, we should keep these templates the same as the source of truth, but for clarity sake let's keep this PR specific just to the changes required to show remote payment links for this cycle and we can create additional tasks to add in any other adjustments at another time

Reply:
Since our public repos are suppose to match with what's in merchantOS, would it be possible to just keep the PR as is so that way we have both bases covered now and we don't have to come back?